workflows/differentialabundance.nf

```diff
@@ -187,8 +187,13 @@ workflow DIFFERENTIALABUNDANCE {
             .map { entry ->
                 def yaml_file = entry[1]
                 def yaml_data = new groovy.yaml.YamlSlurper().parse(yaml_file)
-                yaml_data.contrasts.collect { contrast ->
-                    tuple('id': contrast.comparison[0])
+                yaml_data.contrasts.collect { constrast ->
+                    if (constrast.containsKey('formula')) {
+                        tuple('id': constrast.id)
```

Comment:
Would it not be simpler to mandate an ID in either case?

Reply:
Maybe add a comment to this effect, and link to the relevant line for maxQuant? Then we might remember to go and fix up maxquant in future so this can be simplified.